src/main/java/uk/gov/hmcts/reform/sscs/personalisation/WithRepresentativePersonalisation.java

```diff
@@ -25,11 +27,18 @@
 
     public Map<String, String> setRepresentativeName(Map<String, String> personalisation, SscsCaseData sscsCaseData) {
         if (NotificationUtils.hasRepresentative(sscsCaseData.getAppeal())) {
-            personalisation.put(AppConstants.REPRESENTATIVE_NAME,
+            personalisation.put(REPRESENTATIVE_NAME,
                     SendNotificationHelper.getRepSalutation(sscsCaseData.getAppeal().getRep(), true));
         }
+        overrideRepNameIfNotificationIsForAnOtherParty(personalisation);
 
         return personalisation;
     }
 
+    private void overrideRepNameIfNotificationIsForAnOtherParty(final Map<String, String> personalisation) {
+        if (personalisation.get(OTHER_PARTY) != null) {
```

Comment:
Do you mind adding a comment to this line just explaining that we didn't want to refactor every notification so we are reusing 'rep name' for other parties otherwise it will look confusing for Developers picking this up in future.

Reply:
Done